### DISCUSSION TITLE
Welcome to RUI3-Best-Practice Discussions!

### DISCUSSION THREAD

**Discussion by beegee-tokyo** [RAKWireless/RUI3-Best-Practice#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by scarter84** [comment]
I am looking at implementing my terminal to track livestock and pets (such as a geofenced dog). I want to set up an esp32 LoRa and some collars that emit gps for our farm. So, any insight into this would be most helpful. I am doing some of the front end development so far. I look forward to seeing what all you have come up with. scarter84.github.io/0411/

**Comment by beegee-tokyo** [comment]
Hi @scarter84 
Sorry, you are in the wrong place. RUI3 supports only STM32WLE5, nRF52 and Apollo3 MCU's.
You might get some feedback in our [RAkwireless Forum](https://forum.rakwireless.com/).
